lib/modules/datasource/orb/index.ts

```diff
@@ -47,7 +48,10 @@ export class OrbDatasource extends Datasource {
     if (!registryUrl) {
       return null;
     }
-    const url = `${registryUrl}graphql-unstable`;
+    const path = registryUrl.endsWith('/')
+      ? 'graphql-unstable'
+      : '/graphql-unstable';
+    const url = `${registryUrl}${path}`;
```

Comment:
```suggestion
    const url = joinUrlParts(registryUrl, 'graphql-unstable');
```
we've util functions for such things

Reply:
Thanks!